indexer/indexer.go

```diff
@@ -136,8 +139,8 @@ func ApplyParameters(query string, params ...interface{}) string {
 			}
 		case hexutil.Uint64:
 			preparedParams[i] = fmt.Sprintf("%v", uint64(value))
-		case types.BlockNonce:
-			preparedParams[i] = fmt.Sprintf("%v", int64(value.Uint64()))
+		case BlockNonce:
```

Comment:
Didn't we end up not using this type?

Reply:
The indexer will be addressed in an upcoming PR.